PS C:\codes\socketio-minimal-demo\server> npm start
Debugger listening on ws://128.0.0.1:55693/bf1d0c11-0cbe-46ab-ade0-7a8a2ee7e793
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.

> server@2.0.0 start
> node .

Debugger listening on ws://127.0.0.1:55697/a78285f8-b6a4-49a1-9ab1-7170829d6932
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
node:events:504
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1330:16)
    at listenInCluster (node:net:1378:12)
    at Server.listen (node:net:1465:7)
    at Object.<anonymous> (C:\codes\socketio-minimal-demo\server\index.js:17:6)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1357:8)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8080
}
Waiting for the debugger to disconnect...